tests/foreman/cli/test_discoveredhost.py

```diff
@@ -130,72 +130,6 @@ def test_positive_pxe_based_discovery(self):
             host = self._assertdiscoveredhost(hostname)
             self.assertIsNotNone(host)
 
-    @pytest.mark.tier3
-    def test_positive_pxe_less_with_dhcp_unattended(self):
-        """Discover a host with dhcp via bootable discovery ISO by setting
-        "proxy.type=proxy" in PXE default in unattended mode.
-
-        :id: a23bd065-8385-4876-aa45-e38470be79b8
-
-        :Setup: Provisioning should be configured
-
-        :Steps: Boot a host/VM using modified discovery ISO.
-
-        :expectedresults: Host should be successfully discovered
-
-        :CaseImportance: Critical
-        """
-        with LibvirtGuest(boot_iso=True) as pxe_less_host:
-            hostname = pxe_less_host.guest_name
-            host = self._assertdiscoveredhost(hostname)
-            self.assertIsNotNone(host)
-
-    @pytest.mark.stubbed
-    @pytest.mark.tier3
-    def test_positive_custom_facts_pxeless_discovery(self):
```

Comment:
there's no difference in listing facts of pxe-based or pxeless discovered host, merging both to a single test:
`test_positive_list_facts`
where both builtin and custom facts are to be checked.